hw/dv/tools/dvsim/sim.mk

```diff
@@ -119,6 +119,14 @@ debug_waves:
 ############################
 ## coverage rated targets ##
 ############################
+# Merge coverage if there are multiple builds.
```

Comment:
This comment seems incorrect?

Reply:
oops, accidentally copied this. Updated.